circleci-example/circleci-docker-artifactory/circle.yml

```diff
@@ -7,7 +7,7 @@ dependencies:
     # Install JFrog CLI
     - curl -fL https://getcli.jfrog.io | sh
     # Configure Artifactory instance with JFrog CLI
-    - ./jfrog rt config --url $ARTIFACTORY_URL --user $ARTIFACTORY_USER --apikey $ARTIFACTORY_PASSWORD --interactive=false
+    - ./jfrog config add --artifactory-url $ARTIFACTORY_URL --user $ARTIFACTORY_USER --apikey $ARTIFACTORY_PASSWORD --interactive=false
```

Comment:
Since the server ID is not provided as an argument, this command prompts for the server ID, which blocks thr CI flow.
I noticed that if ```interactive=false``` is not provided, and the CI env var is set to true, the flow is not blocked, and a sefver is set with ```local``` as the server ID, which may be a bug in jfrog-cli.
This seem to apply for all examples.

Reply:
`interactive=false` works on my machine. I added `--interactive=false` to all examples to cover issues whereby some CI servers may not configure the `CI=true` environment variable.